packages/yoastseo/src/scoring/assessments/seo/ProductIdentifiersAssessment.js

```diff
@@ -67,7 +91,7 @@ export default class ProductIdentifiersAssessment extends Assessment {
 	isApplicable( paper ) {
 		const customData = paper.getCustomData();
 		// Product should either be a simple product with a price, or a product with variants.
```

Comment:
Maybe we don't need this comment anymore?

Reply:
agreed